docs/zh-CN/extending-doris/hive-bitmap-udf.md

```diff
@@ -0,0 +1,102 @@
+---
+{
+    "title": "Hive Bitmap UDF",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Hive UDF
+
+ Hive Bitmap UDF 提供了在 hive 表中生成 bitmap 、bitmap 运算等 UDF，Hive 中的 bitmap 与 Doris bitmap 完全一致 ，Hive 中的 bitmap 可以通过 spark bitmap load 导入 doris
+ 
+ 主要目的：
+  1. 减少数据导入 doris 时间 , 除去了构建字典、bitmap 预聚合等流程；
+  2. 节省 hive 存储 ，使用 bitmap 对数据压缩 ，减少了存储成本；
+  3. 提供在 hive 中 bitmap 的灵活运算 ，比如：用户留存、流失等人群 ，计算后的 bitmap 也可以直接导入 doris；
+
+## 使用方法
+
+### 在 Hive 中创建 Bitmap 类型表
+
+```sql
+
+-- 例子：创建 Hive Bitmap 表
+CREATE TABLE IF NOT EXISTS `hive_bitmap_table`(
+  `k1`   int       COMMENT '',
+  `k2`   String    COMMENT '',
+  `k3`   String    COMMENT '',
+  `uuid` binary    COMMENT 'bitmap'
+) comment  'comment'
+
+-- 例子：创建普通 Hive 表
+CREATE TABLE IF NOT EXISTS `hive_table`(
+    `k1`   int       COMMENT '',
+    `k2`   String    COMMENT '',
+    `k3`   String    COMMENT '',
+    `uuid` int       COMMENT ''
+) comment  'comment'
+```
+
+### UDF 使用：
+
```

Comment:
1. Please add english document
2. Are these commands executed in hive, or in doris?